pkg/api/http/server.go

```diff
@@ -174,7 +174,7 @@ func (s *server) StartNonBlocking() error {
 		s.setupRoutes(publicR, s.api.PublicEndpoints())
 
 		healthServer := &http.Server{
-			Addr:              fmt.Sprintf(":%d", *s.config.PublicPort),
+			Addr:              fmt.Sprintf("%s:%d", s.config.PublicListenAddress, *s.config.PublicPort),
```

Comment:
The old code was omitting the listening address while the new code is sending "0.0.0.0" instead. Does it mean that we now will only listen to IPv4 addresses while before it would listen also on IPv6? Can this be validated? I am asking because maybe the default value should probably be empty string to keep backwards compatibility.

Reply:
Thanks @artursouza, good catch! I've updated the default listen addresses CLI flags to be empty strings which will be the same value as before.